go

```diff
@@ -34,6 +34,7 @@ def_command :update_gems, 'Update Ruby gems' do |gems = []|
 end
 
 def_command :test, 'Execute automated tests' do |args = []|
+  exec_cmd 'bundle exec rake test'
```

Comment:
Is this so that projects that include this gem get the test for free with their ./go test? Because `bundle exec rspec` already picks up the test within this branch.